app/services/submission/AdminBlockBuilder.scala

```diff
@@ -39,7 +39,8 @@ class AdminBlockBuilder @Inject()(attachmentsService: AttachmentsService) {
             "customerStatus" -> MTDfB,
             conditional(List(NETP, NonUkNonEstablished).contains(eligibilityData.partyType))(
               "overseasTrader" -> true
-            )
+            ),
+            conditional(vatScheme.business.exists(_.welshLanguage.exists(_ == true)))("welshLanguage" -> true)
```

Comment:
This can be changed to be present always defaulting to false, waiting for confirmation from business.